File: src/main/java/com/natia/secretmod/hooks/KeyboardInputHook.java
package com.natia.secretmod.hooks;

import com.natia.secretmod.features.altC.QuickTab;
import net.minecraftforge.fml.common.eventhandler.EventPriority;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
import net.minecraftforge.fml.common.gameevent.InputEvent;
import org.lwjgl.input.Keyboard;

public class KeyboardInputHook {
    @SubscribeEvent(priority = EventPriority.HIGHEST)
    public void onKey(InputEvent.KeyInputEvent event) {
    }
}

File: src/main/kotlin/natia/skytweaks/hooks/KeyboardInputHook.kt
package natia.skytweaks.hooks

import natia.skytweaks.features.dungeons.NotifyCrush
import natia.skytweaks.features.waypoints.GlobalWaypoints
import net.minecraftforge.fml.common.eventhandler.EventPriority
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
import net.minecraftforge.fml.common.gameevent.InputEvent
import org.lwjgl.input.Keyboard

class KeyboardInputHook {
    @SubscribeEvent(priority = EventPriority.HIGHEST)
    fun onKey(event: InputEvent.KeyInputEvent) {
        if (!Keyboard.getEventKeyState()) return
        GlobalWaypoints.instance.key()
        NotifyCrush.instance.key()
    }
}
